pkg/util/net/ip/ip.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+package ip
+
+import (
+	"io/ioutil"
+	"net"
+
+	netutils "k8s.io/utils/net"
+)
+
+const (
+	// sysfs
+	DISABLE_IPV6_PATH = "/proc/sys/net/ipv6/conf/default/disable_ipv6"
+
+	IPV4_LOOPBACK_ADDRESS = "127.0.0.1"
+)
+
+// GetIPZeroAddress returns INADDR_ANY or INADDR6_ANY (according sysctl disable_ipv6)
+func GetIPZeroAddress() string {
```

Comment:
nit: Seems it is encouraged to avoid `Get` prefixes (I personally dislike get*, that is why I point this out).
ref: https://golang.org/doc/effective_go.html#Getters

Not a must at the moment.